src/reducers/SupplierCreditReducer.js

```diff
@@ -68,9 +68,11 @@ export const SupplierCreditReducer = (state = initialState(), action) => {
 
       const item = UIDatabase.get('Item', itemId);
 
+      const query = 'numberOfPacks > 0 && supplier != null && supplier.isVisible == true';
```

Comment:
Is the `supplier != null` needed here (`supplier.isVisible == true` only if `supplier != null`)?

Reply:
wowow 
108b0a6 
![embares](https://user-images.githubusercontent.com/35858975/74715226-f2c53480-5290-11ea-9d6a-07e1bb47afde.gif)